src/Moat/Types.hs

```diff
@@ -255,6 +260,16 @@ data Protocol
   deriving stock (Eq, Read, Show, Generic)
   deriving stock (Lift)
 
+-- | Kotlin newtype declaration syntax.
+--   The original syntax is "inline class".
+--   Kotlin 1.5 introduced "value class", which
+--   requires a 'JvmInline' annotation.
```

Comment:
Is this a requirement for the user to know about? Or can we add this annotation for the user, without requiring them to remember it?

Reply:
Add this to documentation when it is available, see #14